vignettes/articles/pipeop_torch.Rmd

```diff
@@ -91,7 +91,7 @@ We use the standard cross-entropy loss, SGD as the optimizer and checkpoint our
 checkpoint = tempfile()
 architecture = architecture %>>%
   po("torch_loss", t_loss("cross_entropy")) %>>%
-  po("torch_optimizer", t_opt("sgd")) %>>%
+  po("torch_optimizer", t_opt("sgd", lr=0.01)) %>>%
```

Comment:
I had to add this to avoid the following error,
```r
> graph$train(my.task)
Erreur dans .__ParamSet__get_values(self = self, private = private, super = super,  : 
  Missing required parameters: opt.lr
This happened PipeOp torch_model_classif's $train()
```